plugins/ExamplePlugin/README.md

```diff
@@ -0,0 +1,9 @@
+This is an example of a PLUGIN structure for ExternalModel(s)
+
+Each plugin folder should be organized as follows:
+- src folder containing the plugin source code (see Manual on how to create one):
+./plugins/PluginName/src
+- tests folder containing the tests that are needed to test the plugin linked with RAVEN:
```

Comment:
Is there any documentation in the RAVEN user-manual on how to structure the plugin?

Reply:
Yes.

There is a section in the ```Advanced Users: Plugin```

Reply:
Plus, in the Wiki

Reply:
OK